Assets/_BForBoss/_Entities/Scripts/PlaceholderObstacleDamageBehaviour.cs

```diff
@@ -0,0 +1,16 @@
+using UnityEngine;
+
+namespace Perigon.Entities
+{
+    public class PlaceholderObstacleDamageBehaviour : MonoBehaviour
```

Comment:
I think it would be nice if we had a `Placeholder` folder under scripts to keep things like this in. For better sorting

Reply:
sure